endpoint/endpointComp/endpoint.c

```diff
@@ -126,21 +126,24 @@ COMPONENT_INIT {
   srand(time(NULL));
 
   device_t* device = ta_device(STRINGIZE(EP_TARGET));
-  device->op->get_key(private_key);
-  device->op->get_device_id(device_id);
-
+  if (device == NULL) {
+    LE_ERROR("Get specific device error");
```

Comment:
If the instance of device is null, should it return here?

Reply:
I think the error message can be changed to `Can not get specific device`.

Reply:
We cannot return inside the Legato component.